xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -669,12 +670,6 @@ void CRenderManager::CreateRenderer()
       m_pRenderer = new CRendererMediaCodec;
 #endif
     }
-    if (m_format == RENDER_FMT_MMAL)
```

Comment:
MMALRender supports RENDER_FMT_MMAL (opaque handle decoded by MMALCodec), RENDER_FMT_YUV (software decode by ffmeg) and RENDER_FMT_BYPASS (omxplayer where renderer doesn't see individual frames).